tesseract_utils.py

```diff
@@ -1,19 +1,21 @@
-import numpy
-
 from classification import classes_from_numpy, classes_to_numpy
 from segmentation import segments_from_numpy, segments_to_numpy
+import sys
```

Comment:
`$python3
Python 3.4.2 (default, Oct  8 2014, 10:45:20) 
[GCC 4.9.1] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import io
>>> io.open is open
True`